### ISSUE TITLE
MOON-177 - Column picker

### ISSUE THREAD

**Issue by jahia-ci** [Jahia/moonstone#930]
> Imported from Jira, on Thu Jan 30, 2025
> Issue: [MOON-177](https://jira.jahia.org/browse/MOON-177) in project: Moonstone
> Reporter: @jahia-ci (Samson Blond [X])
> Assignee: _None found in Jira, making reporter the assignee in GitHub_
> Created: Fri Oct 2, 2020, closed: Tue Jul 5, 2022
> Status: Closed (resolution: Won't Do)
>
> **Links**:
> - depends on: [MOON-171 - [Design] Table Component](https://jira.jahia.org/browse/MOON-171) (Closed)
>
